scripts/dev/build_release.py

```diff
@@ -100,6 +100,72 @@ def call_tox(
         env=env, check=True)
 
 
+def fixup_qtwebengine_sandboxing():
+    """WORKAROUND for PyInstaller problems with sanboxing on macOS.
+
+    https://github.com/pyinstaller/pyinstaller/pull/6903
+    https://github.com/qutebrowser/qutebrowser/issues/7278
+    """
+    qute_bundle = pathlib.Path('dist') / 'qutebrowser'
+    app_bundle = pathlib.Path('dist') / 'qutebrowser.app'
+
+    print(f"Attempting to fix up QtWebEngine in .app bundle: {app_bundle}")
+
+    app_res = app_bundle / "Contents" / "Resources"
+    qt_resdir = app_res / "PyQt6" / "Qt6" / 'lib' / 'QtWebEngineCore.framework' / 'Resources'
+
+    qute_lib_dir = qute_bundle / "PyQt6" / "Qt6" / 'lib' / 'QtWebEngineCore.framework' / 'Resources'
+
+    for resource in qt_resdir.iterdir():
+        if resource.suffix != ".pak":
+            continue
```

Comment:
```suggestion
    for resource in qt_resdir.glob("*.pak"):
```

should do the same without needing a manual check.